src/robot/conf/languages.py

```diff
@@ -170,3 +170,265 @@ class Fi(Language):
+    comment_headers = {'Comentário', 'Comentários'}
+    library = 'Biblioteca'
+    resource = 'Recurso'
+    variables = 'Variável'
+    documentation = 'Documentação'
+    metadata = 'Metadados'
+    suite_setup = 'Inicialização de Suíte'
+    suite_teardown = 'Finalização de Suíte'
+    test_setup = 'Inicialização de Teste'
+    test_teardown = 'Finalização de Teste'
+    test_template = 'Modelo de Teste'
+    test_timeout = 'Tempo Limite de Teste'
+    force_tags = 'Forçar Etiquetas'
+    default_tags = 'Etiquetas por Defeito'
+    tags = 'Etiquetas'
+    setup = 'Inicialização'
+    teardown = 'Finalização'
+    template = 'Modelo'
+    timeout = 'Tempo Limite'
+    arguments = 'Argumentos'
+    return_ = 'Retornar'
+    bdd_prefixes = {'Dado', 'Quando', 'Então', 'E', 'Mas'}
+
+from robot.conf import Language
+class PT-br(Language):
+    setting_headers = {'Configuração', 'Configurações'}
+    variable_headers = {'Variável', 'Variáveis'}
+    test_case_headers = {'Caso de Teste', 'Casos de Teste'}
+    task_headers = {'Tarefa', 'Tarefas'}
+    keyword_headers = {'Palavra-Chave', 'Palavras-Chave'}
+    comment_headers = {'Comentário', 'Comentários'}
+    library = 'Biblioteca'
+    resource = 'Recurso'
+    variables = 'Variável'
+    documentation = 'Documentação'
+    metadata = 'Metadados'
+    suite_setup = 'Configuração da Suíte'
+    suite_teardown = 'Finalização de Suíte'
+    test_setup = 'Inicialização de Teste'
+    test_teardown = 'Finalização de Teste'
+    test_template = 'Modelo de Teste'
+    test_timeout = 'Tempo Limite de Teste'
+    force_tags = 'Forçar Etiquetas'
+    default_tags = 'Etiquetas Padrão'
+    tags = 'Etiquetas'
+    setup = 'Inicialização'
+    teardown = 'Finalização'
+    template = 'Modelo'
+    timeout = 'Tempo Limite'
+    arguments = 'Argumentos'
+    return_ = 'Retornar'
+    bdd_prefixes = {'Dado', 'Quando', 'Então', 'E', 'Mas'}
+
+class TR(Language):
+    setting_headers = {'Setting', 'Ayarlar'}
+    variable_headers = {'Variable', 'Değişkenler'}
+    test_case_headers = {'Test Case', 'Test Cases'}
+    task_headers = {'Task', 'Görevler'}
+    keyword_headers = {'Keyword', 'Keywords'}
+    comment_headers = {'Comment', 'Comments'}
+    library = 'Kütüphane'
+    resource = 'Kaynak'
+    variables = 'Değişken'
+    documentation = 'Dokümentasyon'
+    metadata = 'Metadata'
+    suite_setup = 'Suite Setup'
+    suite_teardown = 'Suite Teardown'
+    test_setup = 'Test Setup'
+    test_teardown = 'Test Teardown'
+    test_template = 'Test Şablonu'
+    test_timeout = 'Test Zaman Aşımı'
+    force_tags = 'Force Tags'
+    default_tags = 'Default Tags'
+    tags = 'Etiketler'
+    setup = 'Kurulum'
+    teardown = 'Teardown'
+    template = 'Şablon'
+    timeout = 'Zaman Aşımı'
+    arguments = 'Argümanlar'
+    return_ = 'Return'
+    bdd_prefixes = {'Diyelim ki', 'Eğer ki', 'O zaman', 'Ve', 'Ancak'}
+
+class UK(Language):
+    setting_headers = {'Setting', 'Settings', 'Settings', 'Settings'}
```

Comment:
Markers that aren't translated could also be omitted altogether. Listing them again shouldn't cause problems, though, so leaving them here is fine as well. I guess it could make it easier for others to propose translations.